src/styles/main.scss

```diff
@@ -0,0 +1,129 @@
+$bodyfont: Roboto, sans-serif;
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0 auto;
+  display: flex;
+  padding: 0;
+  gap: 57px;
+  font-family: $bodyfont;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+
+  &_imac {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &_name {
+    color: #060b35;
+    font-family: $bodyfont;
+    font-weight: 500;
+    font-size: 12px;
+    margin: 0 16px 4px;
+    line-height: 18px;
+  }
+
+  &_code {
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0 84px 16px 16px;
+    color: #616070;
+  }
+
+  &_revievs {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 0 16px 24px;
+  }
+
+  &_coments {
+    color: #060b35;
+    margin: 0;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &_price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 16px 16px;
+  }
+
+  &_text {
+    margin: 0;
+    color: #616070;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &_dollars {
+    margin: 0;
+    color: #060b35;
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+  }
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('../images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url('../images/star-active.svg');
+}
+
+.buy {
+  justify-content: center;
+  box-sizing: border-box;
+  align-items: center;
+  text-transform: uppercase;
+  line-height: 16px;
+  font-size: 14px;
+  font-weight: 700;
+  text-align: center;
+  width: 166px;
+  height: 40px;
+  background-color: #00acdc;
+  color: #fff;
+  text-decoration: none;
+  border-radius: 5px;
+  margin: 0 16px 16px;
+  padding: 12px 70px;
+  transition: 0.3s;
+
+  &:hover {
+    background-color: #fff;
+    color: #00acdc;
+    border: 1px solid #00acdc;
+  }
```

Comment:
The `.buy` block appears to be a standalone component and should have its own SCSS file. Additionally, the padding value `70px` seems unusually large, please verify it matches the design.